mapswipe_workers/mapswipe_workers/utils/tasks_to_geojson.py

```diff
@@ -0,0 +1,109 @@
+from osgeo import ogr
```

Comment:
What is the usecase for this module?

It is not been called from any other module. If it is only executed manually then it should be in `mapswipe_workers/scripts/`

Reply:
ogr is used in `tasks_to_geoJson()` to create geojson file for the tasks